### PR TITLE
File paths to use decode rather than a string replace

### PR DESCRIPTION
**Pull request by gavD**
In response to issue #4

### CODE REVIEW

File: src/main/java/com/boxuk/jenkins/jslint/LintRunner.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public Properties call() {
             if(res == null) {
                 this.getClass().getResource("JSLint/jslint.js");
             }
-            String file = res.toString().replaceAll("file:", "");
+            String file = java.net.URLDecoder.decode(res.toString());
```
Comment (athieriot) [inline src/main/java/com/boxuk/jenkins/jslint/LintRunner.java:105]: You can probably import "java.net" but that does not really matter

Comment (athieriot) [inline src/main/java/com/boxuk/jenkins/jslint/LintRunner.java:105]: Actually, it seems that the decode(string) method is deprecated? http://stackoverflow.com/questions/6519746/url-decode-in-java-6

Comment (athieriot) [inline src/main/java/com/boxuk/jenkins/jslint/LintRunner.java:105]: Seems to be used as

```
  URLDecoder.decode(res.toString(), "UTF-8");
```

Comment (athieriot) [inline src/main/java/com/boxuk/jenkins/jslint/LintRunner.java:105]: You can also apparently use:

```
  Charset.defaultCharset()
```

To get the Charset of the plateform but haven't tried that

**Comment by gavD** [comment]
Thanks @athieriot - I've gone with UTF-8 - can this be merrrrged?

**Comment by athieriot** [comment]
Feel free :p

**Pull request merged**